src/api/v2/player.py

```diff
@@ -13,7 +13,7 @@
 
 @router.get("/players/score", response_model=list[KCResponse])

```

Comment:
can we rename to /players/report/score

Reply:
[str between 1-13 and at least 1 str, player/report/score](https://github.com/Bot-detector/public-api/pull/25/commits/1cf961c360e63de882247f3357a5f881cc2684ea)